plugin/web/components/layout/WdHeader.tsx

```diff
@@ -0,0 +1,38 @@
+import { CloseOutlined } from "@ant-design/icons";
+import { Button, Menu, MenuProps, Row } from "antd";
+import { Header } from "antd/lib/layout/layout";
+import { MenuInfo } from "rc-menu/lib/interface";
```

Comment:
Why do you need rc-menu's MenuInfo type? Doesn't seem like you are using rc-menu for anything else